.github/workflows/hardware-long.yml

```diff
@@ -109,12 +115,24 @@ jobs:
           echo "SMC_BOARD=$SMC_BOARD" >> "$GITHUB_ENV"
           echo "BMC_BOARD=$BMC_BOARD" >> "$GITHUB_ENV"
 
+      - name: Select existing firmware bundle
+        run: |
+          case "${{ matrix.config.board }}" in
+            p100) FW_PATH=firmware/p100.fwbundle;;
+            p100a) FW_PATH=firmware/p100a.fwbundle;;
+            p150a) FW_PATH=firmware/p150a.fwbundle;;
+            *) echo "Unknown board type"; exit 1;;
+          esac
+          echo "FW_PATH=$FW_PATH" >> $GITHUB_ENV
+
       - name: Flash Firmware
         working-directory: zephyr
         run: |
           # This needs to be added to the github runner
           export PATH=$PATH:/opt/SEGGER/JLink/
 
+          cp $FW_PATH /opt/tenstorrent/fw/stable/
```

Comment:
We definitely should not overwrite firmware generated in this PR what is considered stable firmware for CI recovery purposes.

I would have to double-check, but I think the target path is only mounted read-only.

Reply:
To flash firmware, all that should be required is `tt-flash --fw-tar fw_pack*.fwbundle --force`